### PR TITLE
Notices up

### PR DESCRIPTION
**Pull request by LIGHT1210**
The changes suggested by @thealphadollar were made and notice is updated

### CODE REVIEW

**Comment by PrashikD** [comment]
All first-hand updates are made and notice board is live

File: index.php [hunk 1]
```diff
@@ -32,11 +32,12 @@
                     <ul id="nav-mobile" class="hide-on-med-and-down right">
                         <li class="active"><a href="index.html" class="waves-effect waves-light">Home</a></li>
                         <li><a href="interiit.php" class="waves-effect">Inter IIT</a></li>
+                        <li><a href="notices.php" class="waves-effect">Notices</a></li>  
                         <li><a href="gc.php" class="waves-effect">GC</a></li>
                         <li><a href="fests.html" class="waves-effect">Fests</a></li>
                         <li><a href="societies.html" class="waves-effect">Societies</a></li>
                         <li><a href="contacts.html" class="waves-effect">Contacts</a></li>
-                        <li><a href="blog/" class="waves-effect">Blog</a></li>
+                        <li><a href="blogpost.html" class="waves-effect">Blog</a></li>
```
Comment (thealphadollar) [inline index.php]: This change is not required!

**Comment by thealphadollar** [comment]
@LIGHT1210 @139Vaibhav @PrashikD Really a splendid job on the changes. Very impressive, thank you for the help :D

**Comment by thealphadollar** [comment]
I've made relevant changes and merged your commits.

**Comment by LIGHT1210** [comment]
Yup sorry it was a mistake... didn't notice when and how it happened.. it
shouldn't be changed..

**Pull request closed without merging**